src/Symfony/Bridge/Doctrine/PropertyInfo/DoctrineExtractor.php

```diff
@@ -208,7 +209,7 @@ private function getMetadata(string $class): ?ClassMetadata
     {
         try {
             return $this->entityManager ? $this->entityManager->getClassMetadata($class) : $this->classMetadataFactory->getMetadataFor($class);
-        } catch (MappingException | OrmMappingException $exception) {
+        } catch (MappingException | OrmMappingException | UnknownEntityNamespace $exception) {
```

Comment:
from the screenshot the exception is `Doctrine\ORM\ORMException` ... am i missing it?

IIUC the explicit way is `!$em->getMetadataFactory()->isTransient($class)` before accessing metadata